201-servicebus-create-topic-subscription-rule/README.md

```diff
@@ -0,0 +1,7 @@
+# 201-servicebus-create-topic-subscription-rule
```

Comment:
@sazeesha See an example of a a good readme that helps the user understand what this template is going to do. https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/sample-README.md

I realize you added what it does but it would also be good for your product to point users to your documentation on these resources. 

For eg., this is a good topic that explains what can be done with a topic https://azure.microsoft.com/en-us/documentation/articles/service-bus-create-topics-subscriptions/ 

There might be other fundamental topics in your documentation that users should know about to learn about Service Bus.